include/Yash.h

```diff
@@ -206,6 +334,28 @@ class Yash {
         YashFunction m_function;
         size_t m_requiredArguments;
     };
+    enum class CtrlSeq {
+        Up,
+        Down,
+        Right,
+        Left,
+        Home,
+        Delete,
+        End,
+        CtrlRight,
+        CtrlLeft
+    };
+    const std::map<std::string, CtrlSeq> SupportedCtrlSeq = {
```

Comment:
Think these should be named with `m_` also and probably be at the bottom since they are variables

Reply:
should be fixed